### PR TITLE
Add basic help showing avilable commands

### PR DESCRIPTION
**Pull request by toddself**
If you don't provide a command to ask for help on, it should show the
list of commands that are available

### CODE REVIEW

**Comment by toddself** [comment]
Should there be more help information in here?  It might help to list where the config file is stored (and that `--version` thing feels "odd" but it's not wrong?)

File: cli/lib/commands/help.js [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
+    const commands = await readdirAsync(__dirname);
+    console.log('Usage: ds <command>');
+    console.log('\nAvailable commands:');
+    console.log(commands.map(cmd => `\t${cmd.split('.')[0]}`).join('\n'));
+    console.log(`\nThe configuration is located at ${userHome}/.entropicrc `);
+  } else {
+    console.log(
+      `help has not been implemented yet for ${command}. You could build it!`
+    );
+    console.log('comands: download, login, publish, whoami');
```
Comment (marabesi) [inline cli/lib/commands/help.js]: commands?

Comment (toddself) [reply cli/lib/commands/help.js]: lol that's a typo from the original file.  fixed :)

**Comment by toddself** [comment]
I just added support for loading help for individual commands from an adjacent text file.  I can make this a separate PR if we want to keep this one small :)

**Pull request merged**